pyproject.toml

```diff
@@ -74,6 +74,11 @@ path = "*/__init__.py"
 # R0903:    Too few public methods
 ignore = "W0611,R0903"
 
+[[tool.pylama.files]]
+path = "*/constants.py"
+# W0611:    Imported but unused
+ignore = "W0611"
+
```

Comment:
The official way to re-export imports is to define an `__all__` for that module with the import listed, so in `constants.py`:
```python
__all__ = [ 
    "__version__",
    "OreSatId",
    "ORESAT_NICE_NAMES",
    "NodeId",
]
```
`*` imports wont pickup the module's internal imports so without `__all__` `from oresat_configs.constants import *` doesn't grab `__version__`. It's still possible to import `__version__` directly of course.

As an added bonus this will stop the `pylint` unused import complaint, and so the special case ignore here can be dropped.